lib/app_theme.dart

```diff
@@ -1,10 +1,78 @@
 import 'package:flutter/material.dart';
 
-class AppTheme {
-  AppTheme._();
+class PrimaryAppTheme {
+  PrimaryAppTheme._();
+  static Color textfieldlabelColor(context) {
+    return Theme.of(context).brightness == Brightness.dark
+        ? Colors.grey[300]
+        : Colors.grey[600];
+  }
+
+  static Color getTextColor(context) {
+    return Theme.of(context).brightness == Brightness.dark
+        ? Colors.white
+        : Colors.black;
+  }
+
+  static Color primaryHeading(context) {
+    return Theme.of(context).brightness == Brightness.dark
+        ? PrimaryAppTheme.primaryColor
+        : Colors.black;
+  }
+
+  static Color boxBg(context) {
+    return Theme.of(context).brightness == Brightness.dark
+        ? PrimaryAppTheme.bgCardDark
+        : PrimaryAppTheme.bgCard;
+  }
+
+  static Color boxShadow(context) {
+    return Theme.of(context).brightness == Brightness.dark
+        ? PrimaryAppTheme.bgCardDark
+        : PrimaryAppTheme.grey;
+  }
+
+  static Color appbarText(context) {
+    return Theme.of(context).brightness == Brightness.dark
+        ? PrimaryAppTheme.primaryColor
+        : Colors.black;
+  }
+
+  static Color drawerIcon(context) {
+    return Theme.of(context).brightness == Brightness.dark
+        ? PrimaryAppTheme.primaryColor
+        : Colors.black;
+  }
+
+  static Color highlightText(context) {
+    return Theme.of(context).brightness == Brightness.dark
+        ? PrimaryAppTheme.primaryColor
+        : PrimaryAppTheme.primaryColorDark;
+  }
+
+  static Color unexpanded_trailing(context) {
+    return Theme.of(context).brightness == Brightness.dark
+        ? Colors.white
+        : Colors.black;
+  }
+
+  static Color expanded_trailing(context) {
+    return Colors.green;
+  }
+
+  static ThemeData getThemeData(context) {
```

Comment:
What is this function's utility? This returns the same stuff irrespective of the condition!